soc/arm/st_stm32/stm32wl/soc.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2020 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file SoC configuration macros for the STM32WL family processors.
+ *
+ * Based on reference manual:
+ *   STM32WLxx advanced ARM ® -based 32-bit MCUs
+ *
+ * Chapter 2.6: Memory organization
```

Comment:
These docstrings don't seem to offer relevant information here

Reply:
I don't challenge that, but it is the way it is done for all other STM32 series.
That could be handle globally in a separate PR.

Reply:
cc: @erwango

Reply:
Indeed this should be cleaned eventually.